db/migrate/20191023180308_subclass_ems_cluster.rb

```diff
@@ -0,0 +1,39 @@
+class SubclassEmsCluster < ActiveRecord::Migration[5.1]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class EmsCluster < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+
+    self.inheritance_column = :_type_disabled
+
+    belongs_to :ext_management_system, :class_name => "SubclassEmsCluster::ExtManagementSystem"
```

Comment:
needs `foreign_key => :ems_id`, since the column name doesn't conventionally match